cmake/dependencies.cmake

```diff
@@ -102,10 +102,10 @@ function(ov_download_tbb)
         # TODO: add target_path to be platform specific as well, to avoid following if
         # build oneTBB 2021.2.1 with Visual Studio 2019 (MSVC 14.21)
         RESOLVE_DEPENDENCY(TBB
-                ARCHIVE_WIN "oneapi-tbb-2021.2.5-win-trim-cfg-simplify.zip"
+                ARCHIVE_WIN "oneapi-tbb-2021.2.5-win-vcruntime.zip"
                 TARGET_PATH "${TEMP}/tbb"
                 ENVIRONMENT "TBBROOT"
-                SHA256 "63bf0992406d69dfb3c402f46466face2388ca55b6678f29072ba304f3622fe7"
+                SHA256 "dc6d5caed18fe4a30e088d1e45f92aac8fa10f542d19019edafa6aeea4465551"
```

Comment:
we can define extra flag in https://github.com/openvinotoolkit/openvino/blob/master/cmake/toolchains/mt.runtime.win32.toolchain.cmake

and download this TBB when flag is true